web/src/i18n/config.ts

```diff
@@ -425,5 +427,13 @@ i18next
         }
       }
     },
+    defaultNS,
     detection: DETECTION_OPTIONS
   })
+
+// export default i18n
```

Comment:
_Minor_: similar comment as in https://github.com/MarquezProject/marquez/pull/2315/files#r1046400526